client/my-sites/checkout/upsell-nudge/professional-email-upsell/index.jsx

```diff
@@ -1,9 +1,225 @@
+			<header className="professional-email-upsell__header">
+				<h3 className="professional-email-upsell__small-title">
+					{ translate( "Hold tight, we're getting your site ready." ) }
+				</h3>
+				<h1 className="professional-email-upsell__title wp-brand-font">
+					{ translate( 'Add Professional Email @%(domain)s', {
+						args: {
+							domain: domainName,
+						},
+						comment: '%(domain)s is a domain name, like example.com',
+					} ) }
+				</h1>
+				<div className="professional-email-upsell__pricing">
+					<span
+						className={ classNames( 'professional-email-upsell__standard-price', {
+							'is-discounted': isEligibleForFreeTrial,
+						} ) }
+					>
+						{ formattedPrice }
+					</span>
+					{ discountBadge }
+				</div>
+			</header>
+			<div className="professional-email-upsell__content">
+				<div className="professional-email-upsell__form">
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Enter email address' ) }
+							<FormTextInputWithAffixes
+								placeholder={ translate( 'Email address' ) }
+								value={ mailboxData.mailbox.value }
+								isError={ hasMailboxError }
+								onChange={ ( event ) => {
+									onMailboxValueChange( 'mailbox', event.target.value.toLowerCase() );
+								} }
+								onBlur={ () => {
+									setMailboxFieldTouched( hasBeenValidated );
+								} }
+								suffix={ `@${ domainName }` }
+							/>
+						</FormLabel>
+						{ hasMailboxError && (
+							<FormInputValidation text={ mailboxData.mailbox.error } isError />
+						) }
+					</FormFieldset>
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Set password' ) }
+							<FormPasswordInput
+								autoCapitalize="off"
+								autoCorrect="off"
+								placeholder={ translate( 'Password' ) }
+								value={ mailboxData.password.value }
+								maxLength={ 100 }
+								isError={ hasPasswordError }
+								onChange={ ( event ) => {
+									onMailboxValueChange( 'password', event.target.value );
+								} }
+								onBlur={ () => {
+									setPasswordFieldTouched( hasBeenValidated );
+								} }
+							/>
+						</FormLabel>
+						{ hasPasswordError && (
+							<FormInputValidation text={ mailboxData.password.error } isError />
+						) }
+					</FormFieldset>
+					<div className="professional-email-upsell__form-buttons">
+						<Button onClick={ handleClickDecline }>{ translate( 'Skip for now' ) }</Button>
+						<Button primary={ true } onClick={ handleAddEmail }>
+							{ translate( 'Add Professional Email' ) }
+						</Button>
+					</div>
+				</div>
+				<div className="professional-email-upsell__features">
+					<h2>{ translate( 'Why get Professional Email?' ) }</h2>
+					<ul className="professional-email-upsell__feature-list">
+						<ProfessionalEmailFeature>
+							{ translate( "Trusted email address that's truly yours" ) }
+						</ProfessionalEmailFeature>
+						<ProfessionalEmailFeature>
+							{ translate( 'Increase your credibility' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Increase your brand credibility' )` *ES Score: 11*